README.rst

```diff
@@ -124,16 +124,20 @@ Depending on your OS and environment, some of these packages may be more challen
 or any issue regarding the above packages, please check their respective documentation.
 
 
-Having trouble with PySCF?
+Quantum Chemistry Packages
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Tangelo can be used without having a quantum chemistry package installed but many algorithms, by default, depend on one being installed.
```

Comment:
I think we need to specify "classical" here, and say what are the main things these package does (compute electronic integrals ? Provide reference classical solvers for HF and post-HF techniques ?)

@AlexandreF-1qbit how would you concisely answer this question ?

Reply:
I changed to Classical Quantum Chemistry Packages. Perhaps there should be a hyphen or something.